core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java

```diff
@@ -288,7 +288,7 @@ private boolean gainPrimacy() throws Exception {
   private void losePrimacy() throws Exception {
     LOG.info("Losing the leadership.");
     if (mServingThread != null) {
-      stopLeaderServing();
+      stopServingGrpc();
```

Comment:
rename stopLeaderServing  to stopServingGrpc as it doesn't STOP the leader serving, but only stops the grpc server.